app/views/discussions/show.html.erb

```diff
@@ -29,7 +29,9 @@
         <%= render partial: 'discussions/description_message', locals: {discussion: @discussion} %>
       <% end %>
       <% @discussion.messages.each do |message| %>
-        <%= render partial: 'discussions/message', locals: {user: message.sender_user, message: message} %>
+        <% unless message.self_deleted? %>
```

Comment:
Not a big deal, but this could be solved with an scope again, like `visible_messages`.
It's up to you now whether to make this a priority of change it afterwards :smile:

Reply:
Did it on a later commit as well.